src/main/kotlin/blackjack/ListCardDeck.kt

```diff
@@ -0,0 +1,18 @@
+package blackjack
+
+class ListCardDeck(
+    private val cards: MutableList<Card>
+): CardDeck {
+
+    override fun drawCard(): Card {
+        checkCardCount()
+        return cards.removeAt(cards.size - 1)
```

Comment:
p3; 컬렉션 함수를 활용할 수도 있겠네요.
```suggestion
        return cards.removeLast()
```